lib/Twig/Environment.php

```diff
@@ -1240,6 +1240,19 @@ public function getTest($name)
             return $this->tests[$name];
         }
 
+        foreach ($this->tests as $pattern => $test) {
```

Comment:
should we first do a `strpos` check on the pattern, to see whether it is actually dynamic ?